demo-java/app/src/main/AndroidManifest.xml

```diff
@@ -18,8 +18,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.example.placesdemo">
 
+  <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

Comment:
This should only be necessary to receive location updates when the app is in the background—I don't think that's necessary in this use case?

Reply:
You're right. I was following the location permissions codelab too literally.